lib/tests/streamlit/caching_test.py

```diff
@@ -31,14 +32,69 @@ def tearDown(self):
         st.caching._cache_info.within_cached_func = 0
         st.caching._cache_info.suppress_st_function_warning = 0
 
-    def test_simple(self):
+    @patch("streamlit.hashing.CodeHasher._get_main_script_directory")
```

Comment:
Should we add the decorator to the rest of the tests in the file that need it, and the ones in `hashing_test.py`?

Maybe there's a before each?

Reply:
Probably.  I'm not sure much would be needed in `hashing_test.py` since that stuff mostly calls the hashing functions directly (i.e. isn't testing the `st.cache` interface).

Reply:
Right, any tests that hash functions in `hashing_test.py` would need it.